torch/_inductor/compile_fx.py

```diff
@@ -242,81 +243,35 @@ def compile_fx_inner(
     boxed_forward_device_index=None,
     user_visible_outputs=frozenset(),
 ):
-    if is_tf32_warning_applicable(gm):
-        _warn_tf32_disabled()
-
     if dynamo_utils.count_calls(gm.graph) == 0:
         return make_boxed_func(gm.forward)
 
-    # lift the maximum depth of the Python interpreter stack
-    # to adapt large/deep models
-    sys.setrecursionlimit(max(sys.getrecursionlimit(), 2000))
-
-    _step_logger()(
-        logging.INFO,
-        "torchinductor compiling "
-        f"{'BACKWARDS' if is_backward else 'FORWARDS'} "
-        f"graph {graph_id}",
-    )
-    V.debug.fx_graph(gm, example_inputs)
-
     if cudagraphs is None:
         cudagraphs = config.triton.cudagraphs
 
-    shape_env = _shape_env_from_inputs(example_inputs)
-
-    # Convert view to reshape in the graph. This is necessary primarily for
-    # layout optimization. Do it unconditionally for uniformity.
-    #
-    # It's needed because when we do layout optimization, an contiguous tensor
-    # in eager mode may becomes a channels last tensor. A view op previously
-    # can be applied to the contiguous tensor may not be able to be applied
-    # on the channels tensor any more. An error like
-    #   RuntimeError: view size is not compatible with input tensor's size and stride
-    #   (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.
-    # will be printed.
-    #
-    # Replace view op to reshape op in this case.
-    # As an example, timm_resnest/botnet26t_256/convnext_base etc. will fail if we don't do this.
-    #
-    # Also this has to be done before FakeTensorProp below to avoid the failed
-    # .view() call.
-    view_to_reshape(gm)
-
-    fake_mode = detect_fake_mode(example_inputs)
-    if not fake_mode:
-        fake_mode = torch._subclasses.FakeTensorMode(allow_non_fake_inputs=True)
-        FakeTensorProp(gm, mode=fake_mode).propagate(*example_inputs)
+    # Inputs to fx_codegen_and_compile
```

Comment:
When would people need to update this list if they add a new config to dynamo or inductor, quick comment might be helpful

Reply:
done